packages/@aws-cdk/aws-kinesis/test/test.stream.ts

```diff
@@ -943,68 +1125,64 @@ export = {
+              ShardCount: 1,
+              RetentionPeriodHours: 24,
+              StreamEncryption: {
+                'Fn::If': [
+                  'AwsCdkKinesisEncryptedStreamsUnsupportedRegions',
+                  {
+                    Ref: 'AWS::NoValue'
+                  },
+                  {
+                    EncryptionType: 'KMS',
+                    KeyId: 'alias/aws/kinesis'
+                  }
+                ]
+              }
             }
           }
         },
-        "Outputs": {
-          "ExportsOutputFnGetAttMyStream5C050E93Arn4ABF30CD": {
-            "Value": {
-              "Fn::GetAtt": [
-                "MyStream5C050E93",
-                "Arn"
-              ]
-            },
-            "Export": {
-              "Name": "stackA:ExportsOutputFnGetAttMyStream5C050E93Arn4ABF30CD"
-            }
-          }
-        }
-      });
-
-      expect(stackB).toMatch({
-        "Resources": {
-          "UserWhoNeedsAccessF8959C3D": {
-            "Type": "AWS::IAM::User"
-          },
-          "UserWhoNeedsAccessDefaultPolicy6A9EB530": {
-            "Type": "AWS::IAM::Policy",
-            "Properties": {
-              "PolicyDocument": {
-                "Statement": [
+        Conditions: {
+          AwsCdkKinesisEncryptedStreamsUnsupportedRegions: {
+            'Fn::Or': [
+              {
+                'Fn::Equals': [
                   {
-                    "Action": [
-                      "kinesis:DescribeStream",
-                      "kinesis:GetRecords",
-                      "kinesis:GetShardIterator"
-                    ],
-                    "Effect": "Allow",
-                    "Resource": {
-                      "Fn::ImportValue": "stackA:ExportsOutputFnGetAttMyStream5C050E93Arn4ABF30CD"
-                    }
-                  }
-                ],
-                "Version": "2012-10-17"
+                    Ref: 'AWS::Region'
+                  },
+                  'cn-north-1'
+                ]
               },
-              "PolicyName": "UserWhoNeedsAccessDefaultPolicy6A9EB530",
-              "Users": [
-                {
-                  "Ref": "UserWhoNeedsAccessF8959C3D"
-                }
-              ]
+              {
+                'Fn::Equals': [
+                  {
+                    Ref: 'AWS::Region'
+                  },
+                  'cn-northwest-1'
+                ]
+              }
+            ]
+          }
+        },
+        Outputs: {
+          ExportsOutputFnGetAttMyStream5C050E93Arn4ABF30CD: {
+            Value: {
+              'Fn::GetAtt': ['MyStream5C050E93', 'Arn']
+            },
+            Export: {
+              Name: 'stackA:ExportsOutputFnGetAttMyStream5C050E93Arn4ABF30CD'
             }
           }
         }
       });
 
       test.done();
     },
-    "fails with encryption due to cyclic dependency"(test: Test) {
+    'fails with encryption due to cyclic dependency'(test: Test) {
```

Comment:
I would try to avoid these changes... makes the PR big for no good reason.

Reply:
fair point, I'd recently configured Prettier to make some cosmetic failures. Will leave these in but avoid it going forward.